README.md

```diff
@@ -56,118 +83,108 @@ The examples below provide codeblocks similar to the example above for various d
+</details>
+
+<details open>
+<summary>Samplers</summary>
 
 * [Warm Starting CMA-ES](./samplers/warm_starting_cma.py)
 
-### Examples of User-Defined Sampler
-
+If you are interested in defining a user-defined sampler, here is an example:
 * [SimulatedAnnealingSampler](./samplers/simulated_annealing_sampler.py)
+</details>
 
-### Examples of Terminator
+<details open>
+<summary>Terminator</summary>
 
 * [Optuna Terminator](./terminator/terminator_simple.py)
 * [OptunaSearchCV with Terminator](./terminator/terminator_search_cv.py)
+</details>
 
-### Examples of Multi-Objective Optimization
-
-* [Optimization with BoTorch](./multi_objective/botorch_simple.py)
-* [Optimization of MLP with PyTorch](./multi_objective/pytorch_simple.py)
-
-### Examples of Visualization
-
-* [Visualizing study](https://colab.research.google.com/github/optuna/optuna-examples/blob/main/visualization/plot_study.ipynb)
-* [Visualizing study with HiPlot](https://colab.research.google.com/github/optuna/optuna-examples/blob/main/hiplot/plot_study.ipynb)
+<details open>
+<summary>Visualization</summary>
 
-### An example to enqueue trials with given parameter values
+* [Visualizing Study](https://colab.research.google.com/github/optuna/optuna-examples/blob/main/visualization/plot_study.ipynb)
+* [Visualizing Study with HiPlot](https://colab.research.google.com/github/optuna/optuna-examples/blob/main/hiplot/plot_study.ipynb)
+</details>
 
-* [Enqueuing trials with given parameters](./basic_and_faq_usages/enqueue_trial.py)
+<details open>
+<summary>Distributed Optimization</summary>
 
-### Examples of aim
-
-* [Tracking optimization process with aim](./aim/aim_integration.py)
-
-### Examples of MLflow
-
-* [Tracking optimization process with MLflow](./mlflow/keras_mlflow.py)
-
-### Examples of Weights & Biases
-
-* [Tracking optimization process with Weights & Biases](./wandb/wandb_integration.py)
-
-### Examples of Hydra
-
-* [Optimization with Hydra](./hydra/simple.py)
-
-### Examples of Distributed Optimization
-
-* [Optimizing on a Dask cluster](./dask/dask_simple.py)
+* [Optimizing on Dask Cluster](./dask/dask_simple.py)
 * [Optimizing on Kubernetes](./kubernetes/README.md)
-* [Optimizing with Ray's joblib backend](./ray/ray_joblib.py)
+* [Optimizing with Ray's Joblib Backend](./ray/ray_joblib.py)
+</details>
 
-### Examples of Reinforcement Learning
+<details open>
+<summary>MLOps Platform</summary>
 
-* [Optimization of Hyperparameters for Stable-Baslines Agent](./rl/sb3_simple.py)
+* [Tracking Optimization Process with aim](./aim/aim_integration.py)
+* [Tracking Optimization Process with MLflow](./mlflow/keras_mlflow.py)
+* [Tracking Optimization Process with Weights & Biases](./wandb/wandb_integration.py)
+* [Optimization with Hydra](./hydra/simple.py)
+</details>
 
-### External projects using Optuna
+<details open>
+<summary>Real-World Optuna Examples by External Projects</summary>
```

Comment:
These links don't provide examples, so how about keeping the original text?

```suggestion
<summary>External projects using Optuna</summary>
```

Reply:
```suggestion
<summary>External Projects Using Optuna</summary>
```